hack/generator/pkg/astmodel/code_generation_context.go

```diff
@@ -142,7 +142,9 @@ func (codeGenContext *CodeGenerationContext) GetAllReachableTypes() Types {
 	for _, pkgImport := range codeGenContext.packageImports.AsSlice() {
 		types, found := codeGenContext.GetTypesInPackage(pkgImport.packageReference)
 		if found {
-			result.AddTypes(types)
+			for k, v := range types {
```

Comment:
Why this change?

Reply:
duplicates were being found. I didn't think much about why.